pyleecan/GUI/Dialog/DMachineSetup/SMSlot/PMSlot10/PMSlot10.py

```diff
@@ -95,6 +146,34 @@ def __init__(self, lamination=None, is_notch=False):
         self.lf_H0.editingFinished.connect(self.set_H0)
         self.lf_Hmag.editingFinished.connect(self.set_Hmag)
 
+    def set_key(self):
+        """Setup the slot key according to the GUI"""
+        if self.g_key.isChecked():
```

Comment:
Faire une liste des variables puis itérer dessus peu importe le cas dans lequel on est

Reply:
Done